### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by v20stepanenko**

### CODE REVIEW

**Review by Lione2009 — APPROVED** [review]

**Review by Lione2009 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+
+#### Project designed on 3-layer architecture:
+
+1. Controllers (Presentation tier)
+2. Services (Application logic tier)
+3. DAO (Data tier)
+
+
+#### TECHNOLOGIES WE USED:
```
Comment (Nick97-git) [inline README.md]:
```suggestion
#### Technologies:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+1. TOMCAT
```
Comment (Nick97-git) [inline README.md]:
```suggestion
1. Tomcat
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#### TECHNOLOGIES WE USED:
+
+1. TOMCAT 
+2. MYSQL  
+3. IoC(inject fields)
+4. JSTL 
+5. JSP 
+6. HTML, CSS 
+7. JDBC 
+8. MAVEN
```
Comment (Nick97-git) [inline README.md]:
```suggestion
8. Maven
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+1. Controllers (Presentation tier)
+2. Services (Application logic tier)
+3. DAO (Data tier)
+
+
+#### TECHNOLOGIES WE USED:
+
+1. TOMCAT 
+2. MYSQL  
+3. IoC(inject fields)
```
Comment (Nick97-git) [inline README.md]: remove this line)

File: README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+5. JSP 
+6. HTML, CSS 
+7. JDBC 
+8. MAVEN
+9. Log4j
+
+
+#### Tips on how to launch the application:
+
+1. Load all Maven dependencies
```
Comment (Nick97-git) [inline README.md]: remove this line

File: README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+3. IoC(inject fields)
+4. JSTL 
+5. JSP 
+6. HTML, CSS 
+7. JDBC 
+8. MAVEN
+9. Log4j
+
+
+#### Tips on how to launch the application:
```
Comment (Nick97-git) [inline README.md]:
```suggestion
#### Steps to launch the application:
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+1. Load all Maven dependencies
+2. You have to install MySQL ,and you can initialize tables with commands from the resources/initialization_db.sql file
+3. Change in /util/ConnectionUtil.java class fields "URL", "user" and "password" to access the created table
+4. In src/core/resources/log4j2.xml in line <File name="LogToFile" File name="path"> you need to change "path" to your absolute path to the file .log
+5. Build project, you can use "mvn clean package" command
```
Comment (Nick97-git) [inline README.md]: remove this line

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- remove redundant / from urls like /drivers/, /cars/, /manufacturers/
- GetMyCurrentCarsController -> Long id -> Long driverId
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> remove /all from /cars/all, /drivers/all, /manufacturers/all url patterns
- AuthenticationFilter -> remove empty line in doFilter() method
- AuthenticationServiceImpl -> logger.warn -> logger.error

**Comment by Nick97-git** [comment]
what about comments related to code?

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**